fid/fid.c

```diff
@@ -316,6 +316,12 @@ M0_INTERNAL int m0_fid_arr_copy(struct m0_fid_arr *to,
 	return M0_RC(0);
 }
 
+M0_INTERNAL void m0_fid_arr_free(struct m0_fid_arr *a)
```

Comment:
@Vinoth2101  arrfid_free() is already defined for this purpose. No need to introduce new function I guess.